django/db/backends/postgresql/schema.py

```diff
@@ -13,6 +13,21 @@ class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
     )
     sql_alter_sequence_type = "ALTER SEQUENCE IF EXISTS %(sequence)s AS %(type)s"
     sql_delete_sequence = "DROP SEQUENCE IF EXISTS %(sequence)s CASCADE"
+    sql_create_sequence = (
+        "CREATE SEQUENCE IF NOT EXISTS %(sequence)s "
+        "AS %(data_type)s "
+        "OWNED BY %(table)s.%(column)s"
+    )
+    sql_set_default_sequence = (
+        "ALTER TABLE %s ALTER COLUMN %s "
+        "SET DEFAULT nextval(pg_get_serial_sequence(%s, %s))"
+    )
+    sql_drop_default = "ALTER TABLE %(table)s ALTER COLUMN %(column)s DROP DEFAULT"
```

Comment:
I wonder if we need this, this isn't all that different to `BaseDatabaseSchemaEditor.sql_alter_column_no_default`

Reply:
I suppose I could combine that with `sql_alter_column`

Reply:
You were right. I ended up removing the default value management code, Django already has a way to do it so there's no need to do it here.